docs/semgrep-secrets/getting-started.md

```diff
@@ -0,0 +1,247 @@
+This section lists the source code managers (SCMs) that Semgrep Secrets supports.
+
+| Source code manager                                     | Semgrep Secrets | PR or MR comments for valid secrets findings |
+| -------------------                                     | --------------- | ----------------------------------------------   |
+| GitHub - Free and Team                                  | ✔️               | ✔️                                                |
+| GitHub - Enterprise Cloud                               | ✔️               | ✔️                                                |
+| GitHub - Enterprise Server                              | ✔️               | ?                                                |
+| GitLab.com (hosted plans) - Free, Premium, and Ultimate | ✔️               | ✔️                                                |
+| GitLab - Self-Managed plans                             | ✔️               | ?                                                |
+
+## Enabling Semgrep Secrets
+
+:::info Prerequisites
+* Semgrep Secrets can only be enabled through Semgrep Cloud Platform (SCP). [<i class="fas fa-external-link fa-xs"></i> Create an account](https://semgrep.dev/login) to enable Semgrep Secrets.
+* You have added or onboarded at least one repository to Semgrep Cloud Platform for scanning. See Starting a SAST and SCA scan on a remote repository.
+:::
+
+1. [<i class="fas fa-external-link fa-xs"></i> Sign in to Semgrep Cloud Platform](https://semgrep.dev/login).
+2. Click **<i class="fa-solid fa-gear"></i> Settings**.
+3. In the Products section, click the **<i class="fa-solid fa-toggle-large-on"></i> Secrets** toggle.
+4. Optional: After enabling Semgrep Secrets, you can trigger a full scan manually through your CI provider or wait for your scheduled Semgrep full scan, typically daily.
+
+## Scanning environments
+
+A scan can be triggered in the following environments:
+
+* You can start a scan **manually** from your CI provider if your CI provider supports running CI jobs on-demand. Refer to your CI provider's documentation for details.
+* You can start a scan **from your CLI**. It is recommended to run CLI scans on non-mainline branches, such as feature branches.
+* When you **open a PR or MR**, Semgrep automatically scans the branch.
+
+### Scanning for secrets through a CI job
+
+After following the steps in [Enabling Semgrep Secrets](#enabling-semgrep-secrets), all Semgrep scans in your CI jobs include secret scanning. There are no additional steps to take.
+
+After each scan, your findings are displayed in **Semgrep Cloud Platform > Secrets**.
+
+:::caution Troubleshooting
+If you do not receive any secrets findings in your scans after enabling Semgrep Secrets, check the following:
+
+* Check the job's log in your CI provider.
+    * Search for the following line or similar to ensure that a secrets scan was included as part of the scan: `Enabled products: Semgrep Code, Semgrep Supply Chain, Semgrep Secrets`
```

Comment:
```suggestion
    * Search for the following line or similar to ensure that a secrets scan was included as part of the scan: `Enabled products: Code, Supply Chain, Secrets`
```

Reply:
![Snag_639f2c](https://github.com/returntocorp/semgrep-docs/assets/20766840/a1998330-4785-4931-8483-f6dc2f3e3b6d)
@raghavjain3 It's the exact CLI output I'm quoting here.